setup.py

```diff
@@ -42,8 +42,6 @@
         "Intended Audience :: Developers",
         "Natural Language :: English",
         "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: py",
-        "Programming Language :: py :: 3",
         "Programming Language :: py :: 3.9",
```

Comment:
Interesting, do we not need these classifiers at all? Are they purely redundant, or is there any value in keeping them here? I'm not entirely sure how these are used...

Reply:
Sorry, didn't fix that one. I will make another update.